### PR TITLE
Changes api that apple incorrectly flagged

### PR DESCRIPTION
**Pull request by steam**
Apple's app store submission automation is flagging internal API as in violation of private Apple API incorrectly. Other folks are having this problem as well. The easiest path to release is changing our internal api to something else. Here goes nothing.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -89,12 +89,12 @@ end
...
   keys: [
-    'ClientKey',
-    'ClientSecret',
+    'OauthKey',
+    'OauthSecret',
     'CrashlyticsKey',
     'Domain',
     'HttpProtocol',
-    'Salt',
+    'SodiumChloride',
```
Comment (colinta) [inline Podfile]: Ha!  Love this.

File: Sources/Networking/APIKeys.swift [hunk 2]
```diff
@@ -41,8 +41,8 @@ public struct APIKeys {
     }
 
     public init() {
-        let key: String = ElloKeys().clientKey()
-        let secret: String = ElloKeys().clientSecret()
+        let key: String = ElloKeys().oauthKey()
+        let secret: String = ElloKeys().oauthSecret()
```
Comment (colinta) [inline Sources/Networking/APIKeys.swift]: Is `ElloKeys().salt()` not used anywhere?

Comment (colinta) [inline Sources/Networking/APIKeys.swift]: `Extensions/CryptoStringExtensions.swift`

Comment (steam) [inline Sources/Networking/APIKeys.swift]: I just had the same thought. It is. I wonder why it works, I must be missing something.

**Comment by mitchellporter** [comment]
LOL at the `SodiumChloride` change. Gotta love the App Store.

**Comment by colinta** [comment]
👍

**Pull request merged**